airflow/providers/microsoft/azure/sensors/wasb.pyi

```diff
@@ -41,3 +42,7 @@ class WasbBlobSensor(BaseSensorOperator):
         check_options: Optional[dict] = None,
         **kwargs,
     ) -> None: ...
+
+# Adding this public module override to suppress Mypy errors stating
+# "airflow.providers.microsoft.azure.sensors.wasb has no attribute 'WasbPrefixSensor'".
+__all__ = ["WasbBlobSensor", "WasbPrefixSensor"]
```

Comment:
I'd love some feedback here on this approach vs. adding the `WasbPrefixSensor` explicitly.

Reply:
Lets' 'remove the .pyi files once and for all: #20936

Reply:
I may have planted the seed so I'll gladly take any heat with you 🙂 

I'll remove this change then.

Reply:
FYI you can put `__all__` in the Python source. It also silences Flake8 so you don’t need `noqa`, which I strongly prefer because `__all__` declares explicitly what are actually expected from the module.

Reply:
Right on, thanks @uranusjr! I'll keep this approach in mind if I ever deprecate modules in the future.